README.md

```diff
@@ -33,6 +33,29 @@ $ docker --help
 
 This should display the list of commands and features.  If it fails with unknown command the installation did not work correctly.
 
+###Several facts which good to know about work with Docker in Linux
+Installation package of Docker contains and installs two independent products: Docker Engine and Docker Desktop.
```

Comment:
`The Docker installation packages consists of two independent products: Docker Engine and Docker Desktop.`